tools/lkl/tests/boot.c

```diff
@@ -112,6 +112,60 @@ int test_getpid(char *str, int len)
 	return TEST_FAILURE;
 }
 
+#ifndef __MINGW32__
+int test_syscall_latency(char *str, int len)
+{
+	struct timespec start, stop;
+	static const int count = 1000;
+	int i;
+	long delta, min = 1000000000, max = -1, avg;
+	int tmp;
+
+	for (i = 0; i < count; i++) {
+		clock_gettime(CLOCK_MONOTONIC, &start);
```

Comment:
Is it possible to use lkl_host_ops to obtain current a wall clock value ? It'll be host independent then.